.travis.yml

```diff
@@ -1,32 +1,59 @@
 language: python
 
+sudo: required
+
 python:
-  - "2.7"
-  - "3.4"
-  - "3.5"
-  - "3.6"
+  - 2.7
+  - 3.5
+  - 3.6
+
+env:
+  - NCO_VERSION=apt
```

Comment:
@kwilcox b/c of https://github.com/nco/pynco/pull/29#issuecomment-303466210 I tried to add the old 4.0.8 version to `conda-forge`.

I could not get  a tarball from sourceforge nor GitHub, so I used the commit hash instead. But that version is too old I was unable to get it to compile. Maybe we should drop it from the matrix here. (And someone should update the debian package!)

my build attempt: https://github.com/ocefpaf/nco-feedstock/tree/nco_4.0.8
the CI results: https://circleci.com/gh/ocefpaf/nco-feedstock/tree/nco_4.0.8